components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/api/client/GradleEnterpriseApiClient.java

```diff
@@ -341,4 +332,10 @@ private static BigDecimal serializationFactorFrom(GradleBuildCachePerformance bu
     private static BigDecimal serializationFactorFrom(MavenBuildCachePerformance buildCachePerformance) {
         return BigDecimal.valueOf(buildCachePerformance.getSerializationFactor());
     }
+
+    public static class UnknownBuildAgentException extends ApiClientException {
```

Comment:
This exception type can be removed.

Reply:
Removed in 0b34bb2bb52df83a2a07d4a0c3b1bf00558f0b0d